tests/src/providers/testqgspostgresconn.cpp

```diff
@@ -203,6 +204,43 @@ class TestQgsPostgresConn: public QObject
       // TODO: add more tests
 
     }
+
+    void sessionRole()
+    {
+      QgsPostgresConn *conn = getConnection();
+      QVERIFY( conn != 0 );
+
+      const QString sql = QStringLiteral( "SELECT SESSION_USER, CURRENT_USER;" );
+
+      QgsPostgresResult result = conn->PQexec( sql );
+      QCOMPARE( result.PQgetvalue( 0, 0 ), "qgis_test_user" );
+      QCOMPARE( result.PQgetvalue( 0, 1 ), "qgis_test_user" );
+
+      const char *connstring = getenv( "QGIS_PGTEST_DB" );
+      if ( NULL == connstring ) connstring = "service=qgis_test";
```

Comment:
Can you fix up all the formatting and old style code here? 
Eg this should be:

```suggestion
      if ( !connstring )
          connstring = "service=qgis_test";
```